src/os/CMakeLists.txt

```diff
@@ -115,16 +120,22 @@ endif()
 
 if(WITH_SPDK)
   include_directories("${CMAKE_SOURCE_DIR}/src/spdk/include")
+  add_custom_target(build_dpdk
```

Comment:
@tchaikov  For using DPDK in SPDK, it will be save to use the submodule pointed by SPDK which will be safe and match the version (since SPDK put some changes on its forked dpdk). I do not think it is necessary or safe to use the installed DPDK ( though it may be used by other purpose.)